core/domain/exp_jobs_one_off.py

```diff
@@ -253,8 +253,8 @@ def reduce(key, values):
 
 
 class ExplorationMigrationAuditJob(jobs.BaseMapReduceOneOffJobManager):
-    """A reusable one-off job for testing exploration migration from the
-    previous exploration schema version to the latest. This job runs the state
+    """A reusable one-off job for testing exploration migration from any
```

Comment:
No, this test still tests migration of the previous exploration schema version to the latest one. (The setup involves bringing the exp to N-1, but the job itself is testing out the N-1 --> N migration.)

Reply:
Changed it now.